src/WebJobs.Script.WebHost/Diagnostics/EventGenerator.cs

```diff
@@ -86,49 +87,56 @@ public void LogFunctionExecutionEvent(string executionId, string siteName, int c
                     WriteEvent(57909, executionId, siteName, concurrency, functionName, invocationId, executionStage, executionTimeSpan, success);
                 }
             }
+        }
+
+        [EventSource(Guid = "a7044dd6-c8ef-4980-858c-942d972b6250")]
+        [SuppressMessage("Microsoft.StyleCop.CSharp.NamingRules", "SA1306:FieldNamesMustBeginWithLowerCaseLetter", Justification = "Need to use Pascal Case for MDS column names")]
+        public class FunctionsSystemLogsEventSource : EventSource
+        {
+            internal static readonly FunctionsSystemLogsEventSource Instance = new FunctionsSystemLogsEventSource();
 
-            [Event(65520, Level = EventLevel.Verbose, Channel = EventChannel.Operational)]
-            public void RaiseFunctionsEventVerbose(string subscriptionId, string appName, string functionName, string eventName, string source, string details, string summary, string hostVersion, string eventTimestamp)
+            [Event(65520, Level = EventLevel.Verbose, Channel = EventChannel.Operational, Version = 1)]
+            public void RaiseFunctionsEventVerbose(string SubscriptionId, string AppName, string FunctionName, string EventName, string Source, string Details, string Summary, string HostVersion, string EventTimestamp)
```

Comment:
Column names match the parameters along with casing. Current MDS configuration uses Pascal casing. Updated this to match existing casing. FunctionsSystemLogsEventSource uses a new GUID.  We will only see logs from this event source once this runtime is deployed. Without the casing change we will see two columns: AppName for old runtimes and appName for new runtimes.